mkdocs.yml

```diff
@@ -1,15 +1,13 @@
 site_name: SimpleLogin Docs
-site_url: https://simplelogin.io/docs/
+site_url: https://simplelogin.io/
 site_description: Receive and send emails anonymously.
 site_author: SimpleLogin
 
 # Repository
 repo_name: simple-login/documentation
 repo_url: https://github.com/simple-login/documentation
-copyright: SimpleLogin is an open-source email alias solution to protect your email address from spam and more.
```

Comment:
the `copyright` here is indeed a bit misleading, it's rather used as the site footer and not like as an actual copyright section.

Reply:
Okay!.